src/main/java/ua/edu/internship/interview/config/SwaggerConfig.java

```diff
@@ -0,0 +1,28 @@
+package ua.edu.internship.interview.config;
+
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.info.Info;
+import io.swagger.v3.oas.models.servers.Server;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.List;
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'java.util.List'
<!-- __reviewdog__:ChBiMTY5MjAwZjU1ODY3YThjEgpjaGVja3N0eWxl -->